ui/src/App.tsx

```diff
@@ -174,13 +178,14 @@ function App() {
             <ListItemIcon>
               <ListIcon />
             </ListItemIcon>
-            <ListItemText primary="List view" />
+            <Link to="/"><ListItemText primary="List view" /></Link>
```

Comment:
Can we get this effect without changing the visual style for follow/unfollowed links? It doesn't seem in keeping the material design menu item style.

Reply:
Done